docs/src/main/asciidoc/_developer-guide/JPA_Reference_Guide.adoc

```diff
@@ -858,6 +858,12 @@ persistence context is closed (e.g. business method ends). Defaults to
 false (entities are cleared after EntityManager invocation) and if set
 to true, the detach is deferred until the context is closed.
 
+|wildfly.jpa.skipquerydetach |Controls whether transaction scoped
```

Comment:
s/whether/whether a/g

Reply:
it may be out of scope, but will see what you think of also making same change nearby in jboss.as.jpa.deferdetach, to be consistent.

Reply:
+1